app/controllers/space_contacts_controller.rb

```diff
@@ -18,7 +18,11 @@ def create
   def update
     return if @space_contact.update(space_contact_params)
 
-    render :edit, status: :unprocessable_entity
+    render turbo_stream: turbo_stream.replace(
+      @space_contact,
+      partial: "space_contacts/form",
+      locals: { space_contact: @space_contact }
+    )
```

Comment:
Kan ta feil om dette, men jeg tror vi har lyst til å tilby både HTML og turbo_stream - sånn at vi fortsatt kan vise noe til brukere som har JS av. 

I turbo handbook så setter de det opp sånn her:  https://turbo.hotwired.dev/handbook/streams

```ruby
# app/controllers/messages_controller.rb
class MessagesController < ApplicationController
  def index
    @messages = Message.all
  end

  def create
    message = Message.create!(params.require(:message).permit(:content))

    respond_to do |format|
      format.turbo_stream do
        render turbo_stream: turbo_stream.append(:messages, partial: "messages/message",
          locals: { message: message })
      end

      format.html { redirect_to messages_url }
    end
  end
end
```

så noe tilsvarende, hvor vi har et format for `format.html` som render den gamle :edit med status: osv, og et `format.turbo_stream` som fikser turbo streamen tror jeg blir rett!

Reply:
Kan vi ta dette i ett eget kort?
Vil kreve en god del endringer.

Reply:
Enig at dette bør gjøres men kan godt bli tatt i et eget kort